tao_compiler/mlir/disc/IR/hlo_disc_ops.td

```diff
@@ -197,4 +197,21 @@ def HLO_QuantizedDynamicConvOp: HLODISC_Op<"quantized_dynamic_conv", [NoSideEffe
   let results = (outs HLO_Tensor:$result);
 }
 
+def HLO_SparseReshapeOp : HLODISC_Op<"sparse_reshape", [NoSideEffect]> {
```

Comment:
better to implement the shape inference interface (`reifyReturnTypeShapes`). like hlo_disc.d2h/h2d ops.

Reply:
done